parsely_raw_data/event.py

```diff
@@ -277,6 +298,12 @@ def to_dict(self):
             event_dict['campaign.__version__'] = self.campaign.__version__
         else:
             event_dict['campaign'] = False
+        if self.flags:
+            event_dict['flags'] = True
+            event_dict['flags.is_amp'] = self.flags.is_amp
```

Comment:
Maybe this logic could be factored into an `EventFlags.to_dict()`, then applied here with `event_dict.update(self.flags.to_dict())`